regression/cbmc-library/Float_lib1/main.c

```diff
@@ -43,10 +43,10 @@ int main()
 
   _Static_assert(__builtin_isnormal(DBL_MIN), "__builtin_isnormal is constant");
 
-  _Static_assert(!__builtin_isinf(0), "__builtin_isinf is constant");
+  _Static_assert(!__builtin_isinf(0.0), "__builtin_isinf is constant");
 
   _Static_assert(
-    __builtin_isinf_sign(0) == 0, "__builtin_isinf_sign is constant");
+    __builtin_isinf_sign(0.0) == 0, "__builtin_isinf_sign is constant");
```

Comment:
Shouldn't that be fixed rather than changing the test?

Reply:
Well, it's the test that was broken: running GCC on this test before the change said:
```
$ gcc regression/cbmc-library/Float_lib1/main.c
regression/cbmc-library/Float_lib1/main.c: In function ‘main’:
regression/cbmc-library/Float_lib1/main.c:46:35: error: non-floating-point argument in call to function ‘__builtin_isinf’
   _Static_assert(!__builtin_isinf(0), "__builtin_isinf is constant");
                                   ^
regression/cbmc-library/Float_lib1/main.c:46:18: error: expression in static assertion is not an integer
   _Static_assert(!__builtin_isinf(0), "__builtin_isinf is constant");
                  ^
regression/cbmc-library/Float_lib1/main.c:49:26: error: non-floating-point argument in call to function ‘__builtin_isinf_sign’
     __builtin_isinf_sign(0) == 0, "__builtin_isinf_sign is constant");
                          ^
regression/cbmc-library/Float_lib1/main.c:49:5: error: expression in static assertion is not an integer
     __builtin_isinf_sign(0) == 0, "__builtin_isinf_sign is constant");
     ^~~~~~~~~~~~~~~~~~~~
```

Reply:
Ah, ok, gcc/clang also error this.